packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -10,7 +10,7 @@ import * as path from 'node:path';
 import { URI } from 'vscode-uri';
 import { WorkspaceContextUtil } from '..';
 import { workspaceUtils } from '../workspaces/workspaceUtils';
-import { createDirectory } from './fs';
+import { createDirectory } from './index';
```

Comment:
yeah, and then if we fix the lint rule these can go back to importing from ./fs.  As is, you've got a circulad dependency between index<->paths